pages/Transactions/index.tsx

```diff
@@ -1,8 +1,89 @@
+import * as React from 'react';
 import Layout from "../../components/Layout/Layout";
+import PropTypes from 'prop-types';
+import { Grid, Button, Tabs, Tab, Box } from "@mui/material";
+import styles from './Transactions.module.css';
 
 const TransactionsPage: React.FunctionComponent = () => {
+
+    function TabPanel(props) {
+        const { children, value, index, ...other } = props;
+      
+        return (
+          <div
+            role="tabpanel"
+            hidden={value !== index}
+            className={styles.box}
+            id={`simple-tabpanel-${index}`}
+            aria-labelledby={`simple-tab-${index}`}
+            {...other}
+          >
+            {value === index && (
+              <Box sx={{ p: 3 }} className={styles.tabbox}>
+                <h3>{children}</h3>
+              </Box>
+            )}
+          </div>
+        );
+      }
+      
+      TabPanel.propTypes = {
```

Comment:
Try defining the props for your functional components like how it's done on `Layout.tsx` by defining it as a new type. This is also for consistency.